ballerina-tests/tests/resources/generated-sources/tool_test_data_type_1/helloWorldWithDependency_pb.bal

```diff
@@ -33,8 +33,8 @@ public isolated client class helloWorldWithDependencyClient {
         } else {
             message = req;
         }
-        var payload = check self.grpcClient->executeSimpleRPC("helloWorldWithDependency/hello", message, headers);
-        [anydata, map<string|string[]>] [result, respHeaders] = payload;
+        var _ = check self.grpcClient->executeSimpleRPC("helloWorldWithDependency/hello", message, headers);
+        [anydata, map<string|string[]>] [_, respHeaders] = payload;
         return {content: <HelloResponse>result, headers: respHeaders};
```

Comment:
I couldn’t find where this `result` come from.

Reply:
Will check